lib/resources/majors.tsx

```diff
@@ -2711,6 +2711,308 @@ const bsMolCell: Major = {
+          criteria: 'AS.080.250[C]',
+        },
+        {
+          description:
+            '<b>Neuroscience: Cellular and Systems I</b> <br /> AS.080.305 Neuroscience: Cellular and Systems I',
+          required_credits: 3,
+          criteria: 'AS.080.305[C]',
+        },
+        {
+          description:
+            '<b>Neuroscience: Cellular and Systems II</b> <br /> AS.080.306 Neuroscience: Cellular and Systems II',
+          required_credits: 3,
+          criteria: 'AS.080.306[C]',
+        },
+      ],
+    },
+    {
+      name: 'Math',
+      required_credits: 12,
+      min_credits_per_course: 3,
+      description:
+        'Must complete Calculus I, Calculus II, and Probability & Statistics.',
+      criteria:
+        'AS.110.10[C]^OR^AS.171.113[C]^OR^Probability and Statistics[N]^OR^Probability & Statistics[N]^OR^EN.553.111[C]^OR^EN.553.112[C]',
+      fine_requirements: [
+        {
+          description:
+            '<b>Probability and Statistics</b> <br />' +
+            'Select one of the following: <br />' +
+            'EN.553.211 ProbStat for Life Sciences <br />' +
+            'EN.553.310 ProbStat for Physical Sciences <br />' +
+            'EN.553.311 ProbStat for Biological Sciences <br />' +
+            'EN.553.111 and EN.553.112 Statistical Analysis I and II',
+          required_credits: 4,
+          criteria:
+            'EN.553.211[C]^OR^EN.553.310[C]^OR^EN.553.311[C]^OR^(EN.553.111[C]^AND^EN.553.112[C])',
+        },
+        {
+          description:
+            '<b>Calculus I</b> <br />' +
+            'AS.110.106 Calculus I (Biology and Social Sciences)' +
+            '<br /> <i>OR</i> <br />' +
+            'AS.110.108 Calculus I (Physical Sciences and Engineering)',
+          required_credits: 4,
+          criteria: 'AS.110.106[C]^OR^AS.110.108[C]',
+        },
+        {
+          description:
+            '<b>Calculus II</b> <br />' +
+            'Select one of the following: <br />' +
+            'AS.110.107 Calculus II (Biology and Social Sciences) <br />' +
+            'AS.110.109 Calculus II (Physical Sciences and Engineering) <br />' +
+            'AS.171.113 Subatomic World',
+          required_credits: 4,
+          criteria: 'AS.110.107[C]^OR^AS.110.109[C]^OR^AS.171.113[C]',
+        },
+      ],
+    },
+    {
+      name: 'Natural Sciences',
+      required_credits: 22,
+      min_credits_per_course: 1,
+      criteria: 'AS Chemistry[D]^OR^AS Physics & Astronomy[D]',
+      description:
+        'Must complete Introductory Chemistry I & II, Organic Chemistry I, and General Physics I & II with their respective labs (except Organic Chemistry).',
+      fine_requirements: [
+        {
+          description:
+            '<b>Chemistry I</b> <br /> AS.030.101 Introductory Chemistry I <br /> AS.030.105 Introductory Chemistry Laboratory I w/ AS.030.105 lab</b>',
+          required_credits: 4,
+          criteria: 'AS.030.101[C]^AND^AS.030.105[C]',
```

Comment:
Might be a good idea to make the theory and lab courses 2 separate fine requirements.